### PR TITLE
Read object store config from environment variables

### PR DESCRIPTION
**Pull request by cjao**
- [ ] I have added the tests to cover my changes.
- [x] I have updated the documentation, VERSION, and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

File: refactor/data/app/api/api_v0/endpoints/fs.py [hunk 1]
```diff
@@ -33,12 +34,27 @@
...
-# )
-# backend = MinioStorageBackend(minio_client)
+# TODO: Improve config management
+storage_backend_selector = os.environ.get("FS_STORAGE_BACKEND", "local")
+local_storage_root = os.environ.get("FS_LOCAL_STORAGE_ROOT", "data")
+bucket_name = os.environ.get("FS_STORAGE_BUCKET", "default")
+minio_endpoint = os.environ.get("MINIO_ENDPOINT", "localhost:9000")
+minio_access_key = os.environ.get("MINIO_ACCESS_KEY", "minioadmin")
+minio_secret_key = os.environ.get("MINIO_SECRET_KEY", "minioadmin")
+minio_use_tls = os.environ.get("MINIO_DISABLE_TLS", "").lower() not in ("true", "1")
```
Comment (scottwn) [inline refactor/data/app/api/api_v0/endpoints/fs.py:44]: Is this the right way to manage config? I thought we had some kind of config manager that wasn't using environment variables.

I'm thinking of this comment from @wjcunningham7 https://github.com/AgnostiqHQ/covalent/pull/215#discussion_r834728936

**Review by scottwn — CHANGES_REQUESTED** [review]

File: refactor/data/app/api/api_v0/endpoints/fs.py (hunk 1 above)
Comment (cjao) [reply refactor/data/app/api/api_v0/endpoints/fs.py:44]: We currently have a config manager, but according to @AlejandroEsquivel 's comment below your referenced comment environment variables will be the way to go after supervisord is integrated.

Comment (cjao) [reply refactor/data/app/api/api_v0/endpoints/fs.py:44]: That said, perhaps the configuration logic might better belong in the modules of the respective backends.

Comment (cjao) [reply refactor/data/app/api/api_v0/endpoints/fs.py:44]: That said, the configuration logic might better belong in the modules of the respective backends.

**Review by scottwn — APPROVED** [review]

**Pull request merged**